### DISCUSSION TITLE
Lighthouse can not resolve its own internal name

### DISCUSSION THREAD

**Discussion by Basti-Fantasti** [slackhq/nebula#1050] [Q&A]
Hi @all

I'm new to Nebula and set up my first network using it.
I just wanted to know if I got some configuration wrong.

I have set up
- a lighthouse node on a VPS with a static IP
- two servers which are connected or find themselves through the lighthouse

The connection is established and on each host I can do a successful ping to the other host:

on host1 I can do:
```bash
ping host2.nebula.domain.net
```
and on host2 I can do it equally:
```bash
ping host1.nebula.domain.net
```

And I get a positive response.

Also on the lighthouse node I can ping both clients with positive responses
```bash
ping host1.nebula.domain.net
ping host2.nebula.domain.net
```

But from both hosts I can not ping the lighthouse node by its name:
```bash
ping lighthouse.nebula.domain.net
```

Any ideas what I have wrong in my config?
Do I need to specify the internal lighthouse name as a static map in the clients config.yml file?

**Comment by Basti-Fantasti** [comment]
Ah in the issues I found the question coming up in issue: #110 
But there seems to be no implementation for the lighthouse by now to announce its own name

**Reply by johnmaguire** [reply]
Correct, you can vote on this issue for now: https://github.com/slackhq/nebula/issues/271
